eng/testing/debug-dump-template.md

```diff
@@ -104,7 +104,7 @@ Now you can debug with WinDbg or `dotnet-dump` as if it was a Windows dump. See
 
 ## ... and you want to debug with LLDB
 
-Install or update LLDB if necessary ([instructions here](https://github.com/dotnet/diagnostics/blob/master/documentation/lldb/linux-instructions.md))
+Install or update LLDB if necessary ([instructions here](https://github.com/dotnet/diagnostics/blob/master/documentation/lldb/linux-instructions.md)). If you install or update LLDB, you must re-run the appropriate `dotnet sos install` command mentioned earlier. This will ensure that LLDB will load SOS for you.
```

Comment:
`sos install` just copies files into your user profile and write a few lines to .lldbinit.

Does installing or updating LLDB writes into your profile? That does not sound right. I have just done `apt-get remote lldb` and `apt-get install lldb` in my WSL Ubuntu to confirm that uninstalling and installing lldb again leaves sos configuration intact.

Reply:
I thought the same at first, although this is fairly harmless. The only thing I can think is lldb was perhaps started before dotnet sos install, so the init file wasn't picked up on start up?

Reply:
Yes, rerunning `sos install` is harmless, but also unlikely to fix anything.

> The only thing I can think is lldb was perhaps started before dotnet sos install, so the init file wasn't picked up on start up?

There are many types of operator errors...

Reply:
> It sounds like if I had missed 'dotnet sos install' earlier that would be consistent with events as well.

Yes, it sounds likely.

> Still, this edit is harmless. I can tune it next time I touch the doc.

The simplest thing to do is to revert this change. I do not think it added anything useful.